scripts/enable-rdp.ps1

```diff
@@ -32,13 +32,15 @@ if($env:appveyor_rdp_password) {
     $password = [Microsoft.Win32.Registry]::GetValue("HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Winlogon", "DefaultPassword", '')
 }
 
-if($ip.StartsWith('172.24.')) {
-    $port = 33800 + ($ip.split('.')[2] - 16) * 256 + $ip.split('.')[3]
-} elseif ($ip.StartsWith('192.168.') -or $ip.StartsWith('10.240.')) {
-    # new environment - behind NAT
-    $port = 33800 + ($ip.split('.')[2] - 0) * 256 + $ip.split('.')[3]
-} elseif ($ip.StartsWith('10.0.')) {
-    $port = 33800 + ($ip.split('.')[2] - 0) * 256 + $ip.split('.')[3]
+if (-not $nonat) {
```

Comment:
it would be better to have variable $nat so you can write 
```
if ($nat) {…}
```

Reply:
Default behavior in 99% of cases is when this variable is not set and custom port is calculated. People who use AppVeyor VMs should not set this `nonat` variable. 
It is only for cases like [this](https://www.appveyor.com/docs/server/#azure-vms) when people connect their own Azure subscription to AppVeyor and set VMs to have public IP. In this case custom port should not be calculated. But it still being calculated because each Azure VM have a private IP.  Though double-negative `if (-not $nonat) {...}` look ugly in code, this would be more intuitive for people using the script.